services/ymax-planner/src/support.ts

```diff
@@ -49,6 +49,54 @@ export const gasLimitEstimates = {
   Wallet: 276_809n,
 };
 
+/**
+ * Average block times for supported EVM chains in milliseconds.
+ *
+ * Sources:
+ * - Ethereum:
+ *   Mainnet ~12s → https://etherscan.io/chart/blocktime
+ *   Sepolia ~12s → https://eth-sepolia.blockscout.com/
+ *
+ * - Arbitrum:
+ *   Mainnet ~0.3s → https://arbitrum.blockscout.com/
+ *   Sepolia ~0.3s → https://arbitrum-sepolia.blockscout.com/
+ *
+ * - Avalanche:
+ *   Mainnet ~2s → https://snowscan.xyz/chart/blocktime
+ *   Fuji ~2s → Didn't find any specific resource for it
+ *
+ * - Base:
+ *   Mainnet ~2.5s → https://base.blockscout.com/stats
+ *   Sepolia ~2s → https://base-sepolia.blockscout.com/
+ *
+ * - Optimism:
+ *   Mainnet ~2s → https://explorer.optimism.io/
+ *   Sepolia ~2s → https://testnet-explorer.optimism.io/
+ */
+const chainBlockTimesMs: Record<CaipChainId, number> = harden({
```

Comment:
Is there a benefit to having so many links?
```suggestion
/**
 * Average block times for supported EVM chains in milliseconds.
 *
 * Mainnet data: https://eth.blockscout.com/ (except Avalanche),
 *   https://chainspect.app/ , https://subnets.avax.network/c-chain
 * Testnet data: https://eth.blockscout.com/ (except Avalanche),
 *   https://subnets-test.avax.network/c-chain
 */
const chainBlockTimesMs: Record<CaipChainId, number> = harden({
```